internal/common/connectionsapi/client_test.go

```diff
@@ -0,0 +1,290 @@
+package connectionsapi
```

Comment:
Can we move this to a test package so that we're testing the public surface of the client
```suggestion
package connectionsapi_test
```